src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -180,7 +181,7 @@ public AttachedRegisterPage openSpecialUserSignUpPage(String wikiURL) {
   public PreferencesPageObject openSpecialPreferencesPage(String wikiURL) {
     getUrl(wikiURL + URLsContent.SPECIAL_PREFERENCES);
```

Comment:
This is out of the scope of this PR, but this method really should not be called `getUrl` (even if driver uses the same nomenclature).

Reply:
great point, reported it in https://wikia-inc.atlassian.net/browse/IRIS-4898